src/cephadm/cephadm

```diff
@@ -6157,14 +6161,10 @@ def _zap_osds(ctx: CephadmContext) -> None:
 
     # list
     mounts = get_container_mounts(ctx, ctx.fsid, 'clusterless-ceph-volume', None)
-    c = CephContainer(
-        ctx,
-        image=ctx.image,
-        entrypoint='/usr/sbin/ceph-volume',
-        envs=ctx.env,
-        args=['inventory', '--format', 'json'],
-        privileged=True,
-        volume_mounts=mounts,
+    c = get_ceph_volume_container(ctx,
+                                  args=['inventory', '--format', 'json'],
```

Comment:
cephadm:6153:5: E124 closing bracket does not match visual indentation
cephadm:6168:5: E124 closing bracket does not match visual indentation